Translation of getting-started.md (#3)
reactjs/pt-PT.reactjs.org: content/docs/getting-started.md

  - "docs/environments.html"
---

Esta página é uma visão geral da documentação e recursos relacionados com React.

**React** é uma biblioteca de JavaScript usada para a construção de interfaces de usuário. Descobre tudo sobre React na [nossa página inicial](/) ou na [página de tutorial](/tutorial/tutorial.html).

---

- [Experimentar React](#try-react)
- [Aprender React](#learn-react)
- [Mantêm-te Informado](#staying-informed)
- [Documentação por Versões](#versioned-documentation)
- [Achas que falta algo?](#something-missing)

## Experimentar React {#try-react}

> O React foi construído desde o início com um método de adaptação em mente, **podes usar React muito ou pouco, conforme as tuas necessidades.** Desde aprenderes React, a adicionares alguma interacção a uma página HTML, ou construíres uma aplicação complexa, as hiperligações nesta secção vão ajudar-te.

Author: @jbarradas podes só alterar a tua sugestão de `contruir` para `construir`?

> O React foi construído desde o início com um método de adaptação em mente, **podes usar React muito ou pouco, conforme as tuas necessidades.** Desde aprenderes React, a adicionares alguma interacção a uma página HTML, ou construíres uma aplicação complexa, as hiperligações nesta secção vão ajudar-te.

Reviewer: Done :p

Tal como qualquer tecnologia, React também tem uma curva de aprendizagem. Com prática e alguma paciência, *vais* aprender rápido.

### Primeiros Exemplos {#first-examples}

A [página inicial do React](/) contém alguns exemplos com um editor de texto online. Mesmo que ainda não saibas nada sobre React, tenta alterar o código e vê como vai afectar o resultado.

### React para Iniciantes {#react-for-beginners}

Caso aches que a documentação está a avançar mais rápido do que te sentes confortável, vê esta [visão geral sobre React da Tania Rascia](https://www.taniarascia.com/getting-started-with-react/). É uma introdução detalhada dos conceitos mais importantes, ideal para iniciantes. Assim que tiveres acabado, volta a esta documentação.

### React para Designers {#react-for-designers}

Caso as tuas origens sejam na área de design, [estes recursos](http://reactfordesigners.com/) são o lugar ideal para começar.

### Recursos de JavaScript {#javascript-resources}

A documentação de React assume alguma familiaridade com a linguagem de programação JavaScript. Não precisas de ser um especialista, mas acaba por ser mais difícil aprender React e JavaScript ao mesmo tempo.

> Recomendamos que acedas a [esta visão geral sobre JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para descobrires qual o teu nível de conhecimento. Deverá demorar entre 30 minutos a uma hora, mas vais sentir-te mais confortável a aprender React.

Reviewer: ```suggestion
Recomendamos que acedas a [esta visão geral sobre JavaScript](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/A_re-introduction_to_JavaScript) para descobrires qual o teu nível de conhecimento. Deverá demorar entre 30 minutos a uma hora, mas vais sentir-te mais confortável a aprender React.
```

>Dica
>

> >Sempre que achares algo confuso em JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) e [javascript.info](http://javascript.info/) são óptimos websites para tirares dúvidas. Também tens [os fóruns de suporte da comunidade](/community/support.html) onde podes pedir ajuda.

Reviewer: ```suggestion
>Sempre que achares algo confuso em JavaScript, [MDN](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript) e [javascript.info](http://javascript.info/) são óptimos websites para tirares dúvidas. Também tens [os fóruns de suporte da comunidade](/community/support.html) onde podes pedir ajuda.
```

Reviewer: O 2º link tem também em pt-PT, mas diz que está incompleto, portanto optei pelo pt-BR.

### Tutorial Prático {#practical-tutorial}

Se preferires **aprender escrevendo código,** acede ao nosso [tutorial prático](/tutorial/tutorial.html). Neste tutorial vamos construir um jogo do galo em React. Podes estar tentado a não o fazer porque não estás a desenvolver jogos -- mas experimenta. As técnicas que vais aprender neste tutorial são fundamentais para construir *qualquer* aplicação em React, e ao seguires o tutorial vai fornecer-te uma compreensão mais profunda.

### Guia Passo a Passo {#step-by-step-guide}

Se preferires **aprender os conceitos passo a passo,** o nosso [guia de conceitos principais](/docs/hello-world.html) é o melhor lugar para começares. A cada novo capítulo vai aumentando o conhecimento apresentado nos capítulos anteriores, de modo a não perderes nada enquanto vais avançando.

### Pensar em React {#thinking-in-react}

> Muitos utilizadores de React dão crédito ao ler [Pensar em React](/docs/thinking-in-react.html) desde que React lhes _fez sentido_. É provavelmente o guia passo a passo mais antigo de React, mas continua a ser relevante.

Reviewer: ```suggestion
Muitos utilizadores de React dão crédito ao ler [Pensar em React](/docs/thinking-in-react.html) como o momento em que o React lhes passou a _fazer sentido_. É provavelmente o guia passo a passo mais antigo de React, mas continua a ser relevante.
```

Reviewer 2: @telmogoncalves veja esta sugestão do @jbarradas

Reviewer 2: @telmogoncalves aqui

### Cursos Recomendados {#recommended-courses}

Algumas pessoas dão-se melhor com livros ou vídeos do que com a documentação oficial. Nós mantemos [uma lista de recursos recomendados](/community/courses.html), alguns deles são gratuitos.

### Conceitos Avançados {#advanced-concepts}

> Assim que te sentires confortável com os [conceitos principais](#main-concepts) e já experimentas-te React por um bocado, talvez tópicos mais avançados sejam do teu interesse. Esta secção vai introduzir-te a poderosas, mas menos comuns funcionalidades como [context](/docs/context.html) e [refs](/docs/refs-and-the-dom.html).

Reviewer: ```suggestion
Assim que te sentires confortável com os [conceitos principais](#main-concepts) e já experimentaste React por um bocado, talvez, tópicos mais avançados sejam do teu interesse. Esta secção vai introduzir-te a poderosas, mas menos comuns funcionalidades como [context](/docs/context.html) e [refs](/docs/refs-and-the-dom.html).
```

Reviewer 2: @telmogoncalves veja esta sugestão do @jbarradas

### Referência API {#api-reference}

> Esta secção da documentação é útil quando queres aprender mais detalhes sobre a API React. Por exemplo, [Referência API `React.Component`](/docs/react-component.html) pode fornecer em detalhe como o `setState()` funciona, e quais são os diferentes métodos de ciclo de vida dos componentes.

Reviewer: ```suggestion
Esta secção da documentação é útil quando queres aprender mais detalhes sobre a API de React. Por exemplo, [Referência API de `React.Component`](/docs/react-component.html) pode fornecer em detalhe como o `setState()` funciona, e qual a utilidade dos diferentes métodos de ciclo de vida dos componentes.
```

### Glossário e FAQ {#glossary-and-faq}

> O [glossário](/docs/glossary.html) contém uma visão geral dos termos mais comuns que vais encontrar na documentação de React. Tens também uma secção de FAQ (_Perguntas e Respostas_) dedicada a pequena perguntas e respostas sobre os tópicos mais comuns, incluindo [fazer uma chamada AJAX](/docs/faq-ajax.html), [componente state](/docs/faq-state.html), e [estrutura de ficheiros](/docs/faq-structure.html).

Reviewer: ```suggestion
O [glossário](/docs/glossary.html) contém uma visão geral dos termos mais comuns que vais encontrar na documentação de React. Tens também uma secção de FAQ (_Perguntas e Respostas_) dedicada a pequenas perguntas e respostas sobre os tópicos mais comuns, incluindo [fazer uma chamada AJAX](/docs/faq-ajax.html), [component state](/docs/faq-state.html), e [estrutura de ficheiros](/docs/faq-structure.html).
```

## Mantêm-te Informado {#staying-informed}

O [blogue do React](/blog/) é a fonte oficial para artigos sobre actualizações por parte da equipa de React. Incluindo notas de versões ou de avisos de substituições, serão publicadas lá primeiro.

Também podes seguir a [conta @reactjs](https://twitter.com/reactjs) no Twitter, mas não vais perder nada importante caso queiras _seguir_ só o blogue.

> Nem todos os lançamentos de versões merecem o seu próprio artigo, mas podes encontrar informação detalhada sobre cada lançamento [no ficheiro `CHANGELOG.md` que se encontra no repositório React](https://github.com/facebook/react/blob/master/CHANGELOG.md), tal como na página de [Lançamentos de Versões](https://github.com/facebook/react).

Reviewer: ```suggestion
Nem todos os lançamentos de versões merecem o seu próprio artigo no blogue, mas podes encontrar informação detalhada sobre cada lançamento [no ficheiro `CHANGELOG.md` que se encontra no repositório React](https://github.com/facebook/react/blob/master/CHANGELOG.md), tal como na página de [Lançamentos de Versões](https://github.com/facebook/react).
```

## Documentação por Versões {#versioned-documentation}

> Esta documentação reflecte sempre a última e mais estável versão de React. Deste a versão 16 de React que podes encontrar documentação para versões anteriores [numa página em separado](/versions). Nota que a documentação para versões anteriores não tem uma actualização instantânea, e não são actualizadas constantemente.

Reviewer: ```suggestion
Esta documentação reflecte sempre a última versão estável de React. Desde a versão 16 de React que podes encontrar documentação para versões anteriores [numa página em separado](/versions). Ressalvamos que a documentação para versões anteriores não tem uma actualização instantânea, e não são actualizadas constantemente.
```

## Achas que falta algo? {#something-missing}

> Se achares que falta algo na documentação ou encontres algo confuso, [abre um _issue_ no repositório da documentação](https://github.com/reactjs/reactjs.org/issues/new) com as tuas sugestões e melhoramentos, podes também enviar um _tweet_ para a [conta @reactjs](https://twitter.com/reactjs). Adoramos ouvir o teu feedback!

Reviewer: ```suggestion
Se achares que falta algo na documentação ou encontrares algo confuso, [abre um _issue_ no repositório da documentação](https://github.com/reactjs/reactjs.org/issues/new) com as tuas sugestões e melhoramentos, podes também enviar um _tweet_ para a [conta @reactjs](https://twitter.com/reactjs). Adoramos ouvir o teu feedback!
```

  - "docs/environments.html"
---

Esta página é uma visão geral da documentação e recursos relacionados com React.

**React** é uma biblioteca de JavaScript usada para a construção de interfaces de usuário. Descubra tudo sobre React na [nossa homepage](/) ou na [página de tutorial](/tutorial/tutorial.html).

---

- [Experimente o React](#try-react)
- [Aprenda React](#learn-react)
- [Mantenha-se Informado](#staying-informed)
- [Documentação por Versões](#versioned-documentation)
- [Acha que falta algo?](#something-missing)

## Experimente o React {#try-react}

> O React foi construído desde o início com um método de adaptação em mente, **você pode usar React muito ou pouco, conforme as suas necessidades.**

Reviewer 2: Ainda não é uma revisão como tal, só quero comentar sobre este cenário. 
Eu prefiro que a malta evite o uso das palavras 'você' ou 'tu' o máximo possível. Assim deixamos a documentação semi-formal. Ficaria assim:

```suggestion
O React foi construído desde o início com um método de adaptação em mente, **podes usar React muito ou pouco, conforme as tuas necessidades.**
```

O que achas @telmogoncalves?

Author: Concordo, mas isso não vai soar estranho ao usar o exemplo que mencionei antes? Aqui usamos **podes**, mas tínhamos falado em usar por exemplo **experimente** em vez de **experimenta**.

Author: Eu prefiro a tua abordagem. Gosto mais de **podes**, **experimenta**, **aprende**, etc.

Reviewer: Sem dúvida. E quando necessário usar entre "tu" e "você", acho que devemos usar o "tu". O "você" é demasiado impessoal.

Reviewer 2: > Concordo, mas isso não vai soar estranho ao usar o exemplo que mencionei antes? Aqui usamos **podes**, mas tínhamos falado em usar por exemplo **experimente** em vez de **experimenta**.

😅 tens razão, acabei por esquecer de mencionar o teu exemplo, então ficamos com o **experimenta** / abordagem "tu"
